modules/persistent-storage-snapshots-starting-controller-provisioner.adoc

```diff
@@ -46,6 +46,12 @@ rules:
     resources: ["volumesnapshotdatas"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ----
+. As the cluster admin, provide the `hostNetwork security context` constraint (SCC):
+----
+# oc adm policy add-scc-to-user hostnetwork -z snapshot-controller-runner
```

Comment:
Just wondering if this command really needs to be run as root (indicated by the # prompt), or if you can use $ instead.